runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -164,6 +164,12 @@
         @Default
         long maxCacheWeight;
 
+        /**
+         * A parameter to tune memory usage vs cache concurrency
+         */
+        @Default
+        int cacheConcurrency = 4;
```

Comment:
Hmm, maybe set this to # cpu ?

Reply:
Can we do this?

Reply:
This is the default for guava. I might just leave it as uninitialized and use the library's defaults instead.

Reply:
Done.